device/arista/x86_64-arista_7060dx5_32/Arista-7060DX5-32-200Gx50-100Gx14/hwsku.json

```diff
@@ -0,0 +1,104 @@
+{
```

Comment:
This should be "200Gx48-100Gx32", i,e, port 1-24 has 2x200G and port 25-32 have 4x100G. please change the file accordingly.

Reply:
Has been changed to 200Gx48-100Gx32. We haven't fully tested the 100G ports after the change for no physical 100G-2 XCVR on the DUT, so far it is only verified that there is no error after the port initialization.